packages/eslint-config-custom/package.json

```diff
@@ -4,10 +4,10 @@
   "main": "index.js",
   "license": "MIT",
   "dependencies": {
-    "eslint-config-next": "^13.4.1",
-    "eslint-config-prettier": "^8.3.0",
+    "eslint-config-next": "14.1.0",
+    "eslint-config-prettier": "8.3.0",
     "eslint-plugin-react": "7.28.0",
-    "eslint-config-turbo": "^1.9.3"
+    "eslint-plugin-turbo": "2.0.3"
```

Comment:
Is this package change intentional? If so we should also update the `eslint-config-custom` to use it as a plugin instead of a config.

Reply:
I had an issue when it requested I'd use the plugin, but rolling it back and just upgrading eslint-config-turbo it's gone now

Probably was a stale lockfile thing